modules/layers/src/text-layer/text-layer.ts

```diff
@@ -607,6 +607,7 @@ export default class TextLayer<DataT = any, ExtraPropsT extends {} = {}> extends
         this.getSubLayerProps({
           id: 'characters',
           updateTriggers: {
+            all: updateTriggers.getText,
             getIcon: updateTriggers.getText,
```

Comment:
You can drop this now that there is an `all` trigger
Same with the `getIconOffsets` trigger

Reply:
Good catch, thanks! f0b94229f